scripts/run.py

```diff
@@ -154,6 +155,12 @@ def stop(self) -> None:
     gemini_concurrency: int = args.gemini_concurrency
     ring_delay_ms: int = args.ring_delay_ms
     enable_rbo : bool = args.enable_rbo
+    language: str = args.language
+
+    supported_languages = ('java', 'rust')
+    if language not in supported_languages:
```

Comment:
Tip: you can use the `choices` parameter for `add_argument` to specify a list of allowed argument values: https://docs.python.org/3/library/argparse.html#choices